src/app/zap-templates/zcl/application-launcher-cluster.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Application Launcher</name>
+    <code>0x050c</code>
+    <define>APPLICATION_LAUNCHER_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
+
+    <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST" type="OCTET_STRING" length="32" writable="false" optional="false">application launcher list</attribute>
+    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="INT8U" min="0x00" max="0xFF" writable="false" default="0x00" optional="true">current app</attribute>
+
+    <command source="client" code="0x00" name="LaunchApp" optional="false">
+      <description>Upon receipt, this SHALL launch the specified app with optional data. The TV Device SHALL launch and bring to foreground the identified application in the command if the application is not already launched and in foreground. The TV Device SHALL update state attribute on the Application Basic cluster of the Endpoint corresponding to the launched application. This command returns a Launch Response.</description>
+      <arg name="application" type="ApplicationLauncherApp" array="true"/>
+      <arg name="data" type="CHAR_STRING"/>
+    </command>
+
+    <command source="server" code="0x00" name="LaunchAppResponse" optional="false">
+      <description>This command SHALL be generated in response to LaunchApp commands.</description>
+      <arg name="status" type="ApplicationLauncherStatus"/>
+      <arg name="data" type="CHAR_STRING"/>
+    </command>
+
+  </cluster>
+      
+  <struct name="ApplicationLauncherApp">
+    <item name="catalogVendorId" type="INT8U"/>
```

Comment:
catalogVendorId seems to be uint16 in the spec

Reply:
Updated.